src/components/cards/StudioRequestsCard.js

```diff
@@ -0,0 +1,118 @@
+import CancelIcon from "@mui/icons-material/Cancel";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  Paper,
+  Stack,
+  Typography,
+} from "@mui/material";
+import React, { useState } from "react";
+
+const StudioRequestsCard = ({
```

Comment:
delete request count. title should refer to the title of the original document being edited. description should refer to the requested fix (i.e grammar fix) and should show on the card. The Dialog should contain the entire edited document (similar to document page but just content (i.e no ToC). lmk if you'd like to call regarding this.